server/src/main/java/com/tencent/rss/server/ShuffleFlushManager.java

```diff
@@ -145,6 +150,7 @@ public void addToFlushQueue(ShuffleDataFlushEvent event) {
 
   private void flushToFile(ShuffleDataFlushEvent event) {
 
+    event.pickStorageIndex(storageBasePaths.length);
```

Comment:
If storage is broken during the shuffle, shuffle data will be located in different location for the same partition?

Reply:
Yes, partition will locate an another storage. Actually, once a storage is broken,  I hope shuffle will fail fast. I hope to implement this function.